app/models/subject.rb

```diff
@@ -11,8 +11,4 @@ def self.sorted_by_number
   def self.with_assessments
     joins(:assessments).distinct
   end
-
-  def to_s
```

Comment:
@derekprior This shouldn't be fixed in here, but isn't `to_s` a built in rails method? It seems odd to me to name a custom method the same as one that's build in.